client/organization.go

```diff
@@ -40,6 +40,10 @@ func (client *ApiClient) Organization() (Organization, error) {
 	return result[0], nil
 }
 
+func (client *ApiClient) OrganizationIdCaching() (string, error) {
```

Comment:
I'm starting to think we should just change organizationId() to OrganizationId()
(it should be easy to "replace all" in the IDE).

You will also need to re-generate go-mock:
`go generate ./...` (see README.md).

Reply:
I agree, wasn't confidant of doing that change but it felt required
[46d3f7a](https://github.com/env0/terraform-provider-env0/pull/456/commits/46d3f7acd4e5f8f146e04018b2ffbc25f81c672b)